### DISCUSSION TITLE
How do I change the "Python 3.10" title?

### DISCUSSION THREAD

**Discussion by RaymonDev** [GitHub30/win11toast#19] [Q&A] [answered]
Hi everyone!!

I've been loooking at the documentation for a while. I have noticed that in every notification it says on top "Python 3.10" or something like that. I there a way to change that text?

Thank you!!!

**Comment by GitHub30** [comment]
![image](https://user-images.githubusercontent.com/12811398/224065741-c9033824-a6cd-4d09-a934-ce96dfceb188.png)

https://learn.microsoft.com/en-us/windows/apps/design/shell/tiles-and-notifications/adaptive-interactive-toasts?tabs=appsdk#attribution-area
https://learn.microsoft.com/en-us/windows/apps/design/shell/tiles-and-notifications/adaptive-interactive-toasts?tabs=appsdk#context-menu-actions

**Reply by GitHub30** [reply]
![image](https://user-images.githubusercontent.com/12811398/224066486-b295c82c-a5c4-449c-9a91-cc9fce45ad79.png)

**Comment by thisal-d** [comment, marked as answer]
```
#use app_id kwarg .
win11toast.toast(app_id="App Name" ,
                           title='Hello'  ,
                           body='Hello World' )


```

![Screenshot 2023-05-19 030117](https://github.com/GitHub30/win11toast/assets/93121062/2797c1db-d6ee-4730-84b6-8388947450b5)

**Reply by RaymonDev** [reply]
Thank you so much!!!
